src/main/thrift/parquet.thrift

```diff
@@ -511,6 +511,33 @@ struct ColumnMetaData {
    * This information can be used to determine if all data pages are
    * dictionary encoded for example **/
   13: optional list<PageEncodingStats> encoding_stats;
+
+  /** Byte offset from beginning of file to bloom filter data**/
+  14: optional i64 bloom_filter_offset;
+}
+
+enum BloomFilterAlgorithm {
```

Comment:
This file does not contain in it enough information to fully reconstruct the algorithm.

Reply:
Hi Jim
When we get bloom filter offset from the column chunk metadata, we can read the header defined to parse the HASH and ALGORITHM, and read BYTES of bitset.  What other information you think we lack here?

Reply:
Imagine that your design document was lost and the parquet-mr patch was lost. Shouldn't thi patch have enough information to support a complete reconstruction of the algorithm, as Ryan Blue mentioned in the call today?

This contains no information about the salt, about WHICH block-based bloom filter is used, the size of the blocks, the encoding of the data that is hashed, and so on.

Reply:
Thanks Jim, I added more info about this.